tests/js/spec/views/eventsV2/utils.spec.jsx

```diff
@@ -244,287 +241,3 @@ describe('getFieldRenderer', function() {
     expect(link.find('StyledDateTime').props().date).toEqual(data.createdAt);
   });
 });
-
-describe('decodeColumnOrderAndColumnSortBy', function() {
```

Comment:
Do we not have any tests for the new behavior?

Reply:
@leedongwei @markstory I've salvaged the tests for `decodeColumnOrderAndColumnSortBy` and refactored it for the `decodeColumnOrder` function that's being used within `EventView`. Can you please check if it's fine?